tunnel-server/src/ssh-server.ts

```diff
@@ -2,12 +2,26 @@ import crypto, { randomBytes } from 'crypto'
 import { FastifyBaseLogger } from 'fastify/types/logger'
 import net from 'net'
 import path from 'path'
-import ssh2 from 'ssh2'
+import ssh2, { ParsedKey, SocketBindInfo } from 'ssh2'
 import { inspect } from 'util'
 
 const idFromPublicSsh = (key: Buffer) =>
   crypto.createHash('sha1').update(key).digest('base64url').replace(/[_-]/g, '').slice(0, 8).toLowerCase()
 
+const getRequestedSocketPath = (info: ssh2.SocketBindInfo) => {
+  const [path, access] = info.socketPath.split('#')
```

Comment:
I would choose an extendable format like
`mypath;auth=profile-key`
this way we can add more stuff here